res/__tc/clang/0_config.sh

```
@@ -0,0 +1,6 @@
+pwd
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)